timemachine/md/minimizer.py

```diff
@@ -59,15 +59,18 @@ def parameterize_system(topo, ff: Forcefield, lamb: float):
             lamb,
         ),
     ]
-    return params_potential_pairs
+    return [p for _, p in params_potential_pairs], [params for params, _ in params_potential_pairs]
 
 
-def bind_potentials(params_potential_pairs):
-    pots = [pot for _, pot in params_potential_pairs]
-    params = [p for p, _ in params_potential_pairs]
-    flat_params = np.concatenate([p.reshape(-1) for p in params])
-    u_impls = [SummedPotential(pots, params).bind(flat_params).to_gpu(precision=np.float32).bound_impl]
-    return u_impls
+def flatten_params(params: List[NDArray]) -> NDArray:
+    return np.concatenate([p.reshape(-1) for p in params])
+
+
+def summed_potential_from_potentials_and_params(
```

Comment:
Could be another PR, but it seems like this would be nice as a general utility. Maybe it could be a class method like `SummedPotential.from_potentials_and_params` or `SummedPotential.from_bound_potentials`?

Reply:
Will follow up with another PR, agree would be a nice utility to have.

Reply:
Playing around with this a bit, `SummedPotential.from_bound_potentials` might be useful and `SummedPotential.from_potentials_and_params` would be identical to `SummedPotential(potentials, params)`.  With this function taking care of the additional need to flatten the params at this stage.

Reply:
> SummedPotential.from_potentials_and_params would be identical to SummedPotential(potentials, params)

Ah, I guess the analog of the function here would return a `BoundPotential[SummedPotential]`, so may not make sense as a class method of `SummedPotential`. Would have to think about this more, but could imagine a class extending `BoundPotential[SummedPotential]` with a `from_potentials_and_params` class method and possibly other utilities for handling flattening/unflattening (in a future PR)